### ISSUE TITLE
Can't use overlay method on powder snow

### ISSUE THREAD

**Issue by pazuchi214** [PepperCode1/Continuity#574]
I'm trying to make a resource pack. I want to use overlay method to make powder snow look spreading, so I wrote a .properties file for it, but it doesn't work. The same .properties file works for snow block. I wonder if this is a bug, or I've made mistakes in the .properties file. The game version is 1.20.1, and mod version is 3.0.0 using connector 1.0.0 beta.46 in forge. 
Here's my .properties file:
<img width="936" alt="properties file" src="https://github.com/user-attachments/assets/6f373f4c-65b3-4abd-92fe-0f66750bfb33" />

**Comment by PepperCode1** [comment]
Non-full blocks cannot apply overlays; this is intended and works the same way in OptiFine. Changing this behavior breaks some existing packs, but I plan to add a new property to allow it in the future.

**Comment by pazuchi214** [comment]
I see! Thank you for your reply. Looking forward to the next update :)

**Comment by DawsonBodenhamer** [comment]
I'm also excited for that update! I have Continuity and I'm working on a connected textures resource pack but running into the same issue with powder snow

**Comment by 123happily** [comment]
Sorry to be a bother- I'd love to see this feature implemented!
For example it would allow smart snow overlays with the snow layers as well; as opposed to making the snow layer model have an overhang (which then applies to snow blocks, not ideal)
I'm trying to find a workaround with conditional model usage but the ability to cast overlays from non-full blocks really would be ideal.
Thanks for all you do!
